protomongo/protomongo_test.go

```diff
@@ -0,0 +1,124 @@
+package protomongo
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/dataform-co/dataform/protos/dataform"
+	"github.com/golang/protobuf/proto"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+var (
+	tests = []struct {
+		name string
+		pb   proto.Message
+		new  func() proto.Message
+	}{
+		{
+			name: "simple message",
+			pb: &dataform.ProjectConfig{
```

Comment:
Seems weird to use / depend on dataform protos. Why not make a testing proto message with every type in it?